references/contributing.md

```diff
@@ -23,13 +23,33 @@ This is the main repository that provides the `deno` CLI.
 If you want to fix a bug or add a new feature to `deno` this is the repository
 you want to contribute to.
 
-Languages: **Rust**, **JavaScript**
+Some systems, including a large part of the Node.js compatibility layer are
+implemented in JavaScript and TypeScript modules. These are a good place to
+start if you are looking to make your first contribution.
+
+While iterating on such modules it is recommended to include
+`--features __runtime_js_sources` in your `cargo` flags. This is a special
+development mode where the JS/TS sources are not included in the binary but read
+at runtime, meaning the binary will not have to be rebuilt if they are changed.
```

Comment:
We should mention that using `rust-analyzer` might require additional configuration, otherwise it will rebuild without this flag in the editor on each change.